core/drivers/crypto/versal/crypto.mk

```diff
@@ -0,0 +1,14 @@
+ifeq ($(CFG_VERSAL_CRYPTO_DRIVER),y)
+# Enable the crypto driver
+$(call force,CFG_CRYPTO_DRIVER,y)
+
+CFG_CRYPTO_DRIVER_DEBUG ?= 0
+$(call force,CFG_CRYPTO_DRV_ACIPHER,y)
+$(call force,CFG_CRYPTO_DRV_ECC,y)
+$(call force,CFG_CRYPTO_DRV_RSA,y)
+$(call force,CFG_CRYPTO_DRV_AUTHENC,y)
+
+# Disable Fault Mitigation: currently affected by
+# the need to fallback to software operations .
+CFG_FAULT_MITIGATION = n
```

Comment:
`CFG_FAULT_MITIGATION ?= n`
or 
`$(call force,CFG_FAULT_MITIGATION,n)`

Reply:
I had tried both options:

1) the first one will not work as crypto.mk runs last and someone else enabled CFG_FAULT_MITIGATION (so it wont force it)

2) returned the following error probably for the same reason
 core/drivers/crypto/versal/crypto.mk:13: *** CFG_FAULT_MITIGATION is set to 'y' (from file) but its value must be 'n'.  Stop.

the simpler choice I see, since this make runs last, is to do as I did.

Reply:
Set `CFG_FAULT_MITIGATION ?= n` in `core/arch/arm/plat-versal/conf.mk`.

Reply:
right...but since the issue is related to crypto I thought it would make sense to set it there.

why is doing that a problem? what is wrong with setting it to n in crypto.mk?

Reply:
btw I dont mind doing what you propose @jforissier, just would like to understand the reason behind it.

Reply:
Sure, see: https://github.com/OP-TEE/optee_docs/pull/182/files

Reply:
ahh, yes, passing CFG_ through the environment or build line..ok thanks!

Reply:
In this _crypto.mk_ file you could still check the value:
```makefile
$(call force,CFG_FAULT_MITIGATION,n,Required for Versal support)
```
 but as @jforissier said, platform config shall enforce the expected value knowing they enable the hw device.